test/evergreen.yml

```diff
@@ -278,12 +280,19 @@ functions:
               git add $branch
               git commit -m "Update auto-generated docs for $branch" \
                         --author="svc-bot-doc-build <svc-wiredtiger-doc-build@10gen.com>"
-              git push --dry-run https://"${doc-update-github-token}"@github.com/wiredtiger/wiredtiger.github.com
             else
               echo "No documentation changes for $branch."
             fi
 
           done
+    - command: shell.exec
```

Comment:
I thought we were doing everything in one shell command? I'm confused.

Reply:
I think it brings two main benefits:
- We can enable verbose for those commands now, easier to debug in case of an issue
- We can isolate the command which prints sensitive information when verbose is enabled
However, it is more lines of code and the flow might not be the best.

Reply:
The cleaner solution and best practice is to write messages to the log containing the information you need to debug in case of a failure. I fail to understand how it is more useful to echo every line of the script, particularly as local variables aren't even being expanded. That said, the engineers are the ones who should determine what log output is most useful for debugging, and are also the ones who need to maintain this file. So Etienne, it's up to you :)